src/app/Fake.Installer.Squirrel/Squirrel.fs

```diff
@@ -72,12 +88,19 @@ let private createSigningArgs (parameters : ReleasifyParams) =
     |> StringBuilder.toText
 
 let internal buildSquirrelArgs parameters nugetPackage =
-    new StringBuilder()
+    let sb = new StringBuilder()
+
+    parameters.AdditionalArguments |> Seq.iter (fun arg -> StringBuilder.appendIfNotNullOrEmpty arg "" sb |> ignore)
+
+    sb
     |> StringBuilder.appendIfNotNullOrEmpty nugetPackage "--releasify="
```

Comment:
I'd recommend updating this to the Arguments/CreateProcess API as the StringBuilder is a bit unsafe

Reply:
I have now added a commit that uses the CreateProcess API and uses only a string option for the args